packages/app/package.json

```diff
@@ -285,7 +286,8 @@
     "@types/react-dom": "^16.8.3",
     "@types/react-helmet": "^5.0.11",
     "@types/react-icons": "2.2.7",
-    "@types/react-instantsearch": "^5.2.3",
+    "@types/react-instantsearch-dom": "^5.2.3",
+    "@types/react-instantsearch-core": "^5.2.3",
```

Comment:
maybe it works if the changes in package.json are undone (it should also work in theory thanks to the transitive dependencies used